osu.Game/Screens/SelectV2/PanelBeatmap.cs

```diff
@@ -96,7 +96,7 @@ private void load(OverlayColourProvider colourProvider)
                             AutoSizeAxes = Axes.Both,
                             Children = new Drawable[]
                             {
-                                starRatingDisplay = new StarRatingDisplay(default, StarRatingDisplaySize.Small)
+                                starRatingDisplay = new StarRatingDisplay(default, StarRatingDisplaySize.Small, animated: true)
```

Comment:
Just to confirm, this is supposed to _always_ animate? Or in other words, you're fine with this?

I'm biased because I know what I'm looking for, but I found this potentially jarring - some panels animate from 3* to 7*, some panels from 3* to 2*, some from 3.5* to 3.5*, etc... All with nomod.

See for example this video (watch the bottom-most set):

https://github.com/user-attachments/assets/2409ab2f-a2e1-45ca-ace5-5b40c7be931a

Also notice how the ruleset icon the `Down's Extra` difficulty flashes yellow briefly.

Reply:
The animations are a bit broken with pooling at the moment, so try and ignore that for now.